vamb/__main__.py

```diff
@@ -1151,6 +1492,39 @@ def main():
         default=1e-3,
         help="learning rate [0.001]",
     )
+
+    # Train predictor arguments
+    pred_trainos = parser.add_argument_group(title="Training options for the taxonomy predictor", description=None)
+
+    pred_trainos.add_argument(
+        "-pe", dest="pred_nepochs", metavar="", type=int, default=100, help="taxonomy predictor epochs [100]"
+    )
+    pred_trainos.add_argument(
+        "-pt",
+        dest="pred_batchsize",
+        metavar="",
+        type=int,
+        default=256,
+        help="starting batch size for the taxonomy predictor [256]",
+    )
+    pred_trainos.add_argument(
+        "-pq",
+        dest="pred_batchsteps",
+        metavar="",
+        type=int,
+        nargs="*",
+        default=[25, 75],
+        help="double batch size at epochs for the taxonomy predictor [25 75]",
+    )
```

Comment:
I wonder if increasing the batchsteps helps for the predictor. We might want to have our student check that out, too, if he has time.